services/api-server/src/simcore_service_api_server/services/catalog.py

```diff
@@ -121,14 +127,15 @@ async def get_service(
         )
         response.raise_for_status()
 
-        service = TruncatedCatalogServiceOut.parse_obj(response.json())
+        service = TruncatedCatalogServiceOut.parse_raw(response.text)
```

Comment:
same here.
actually this could run in an async executor in order to ensure it does not freeze the event loop